### ISSUE TITLE
Plex Transcoder ENOENT

### ISSUE THREAD

**Issue by drouarb** [UnicornTranscoder/UnicornTranscoder#31]
Crash on spawn Plex Transcoder even if the file is here
```
Sep 16 21:34:27 rick unicorn-transcoder[11181]: 2018-09-16T21:34:27.549Z transcoder Spawn hkmt5sht68gao6xdm1jfqxu6
Sep 16 21:34:27 rick unicorn-transcoder[11181]: events.js:183
Sep 16 21:34:27 rick unicorn-transcoder[11181]:       throw er; // Unhandled 'error' event
Sep 16 21:34:27 rick unicorn-transcoder[11181]:       ^
Sep 16 21:34:27 rick unicorn-transcoder[11181]: Error: spawn /root/unicorn/UnicornTranscoder/Resources/Plex Transcoder ENOENT
Sep 16 21:34:27 rick unicorn-transcoder[11181]:     at _errnoException (util.js:1022:11)
Sep 16 21:34:27 rick unicorn-transcoder[11181]:     at Process.ChildProcess._handle.onexit (internal/child_process.js:190:19)
Sep 16 21:34:27 rick unicorn-transcoder[11181]:     at onErrorNT (internal/child_process.js:372:16)
Sep 16 21:34:27 rick unicorn-transcoder[11181]:     at _combinedTickCallback (internal/process/next_tick.js:138:11)
Sep 16 21:34:27 rick unicorn-transcoder[11181]:     at process._tickCallback (internal/process/next_tick.js:180:9)
```

**Comment by drouarb** [comment]
Ok the problem come from the NodeJS message, it just means that the directory used as [cwd](https://github.com/UnicornTranscoder/UnicornTranscoder/blob/4ba45f68b1693c65c80067c04e749c7fa172a98f/core/transcoder.js#L119) does not exists.

**Comment by Maxou44** [comment]
It was fixed in 2.0.0 no?

**Comment by drouarb** [comment]
Normally yes
